src/definitions/modules/slider.js

```diff
@@ -353,13 +353,13 @@ $.fn.slider = function(parameters) {
                 thumbSmoothVal = module.determine.smoothValueFromEvent(event)
               ;
               if(!$currThumb.hasClass('second')) {
-                if(settings.preventCrossover) {
+                if(module.is.range() && settings.preventCrossover) {
```

Comment:
I think that evaluating boolean flag `settings.preventCrossover` is faster than `module.is.range()`.

So I propose to flip them like `if(module.is.range() && module.is.range()) {`

Reply:
`module.is.range()` is only checking for a class https://github.com/ColinFrick/Fomantic-UI/blob/fix/1231/src/definitions/modules/slider.js#L473 the performance benefit from switching these would be basically invisible but change as you wish @ColinFrick

Reply:
Done